src/docs/asciidoc/getting-started.adoc

```diff
@@ -6,163 +6,47 @@ include::{docdir}/variables.adoc[]
 
 = Getting Started
 
-== System requirements
 
-// PREREQUISITES
-include::{docdir}/common/prerequisites/prerequisites.adoc[]
 
-== Create your first application
+== System requirements
+include::{docdir}/getting-started/requirements/system-requirements.adoc[leveloffset=+1]
 
-=== From the command line
 
-include::{docdir}/common/cli/create-project.adoc[leveloffset=+2]
 
+== Create your first application
 
-[TIP]
-====
-See <<developer-manual.adoc#_zetapush_account_management, ZetaPush account management>> to learn how to manage your account and your applications.
-====
+=== From the command line
+include::{docdir}/getting-started/init/create-project.adoc[leveloffset=+2]
 
 === Generated project
+include::{docdir}/getting-started/init/generated-project.adoc[leveloffset=+2]
 
```

Comment:
I think we need to say "Generated **application**" to avoid lack of understanding.

Likewise the titles"create-project.adoc" and "generated-project.adoc"